pep-0480.txt

```diff
@@ -276,7 +280,7 @@ Files, and Key Management) cover the cryptographic components of the developer
 release process.  That is, which key type PyPI supports, how keys may be
 stored, and how keys may be generated.  The two subsections that follow the
 first three discuss the PyPI modules that SHOULD be modified to support TUF
-metadata.  For example, Twine and Distutils are two projects that SHOULD be
+metadata.  For example, Twine and Distlib are two projects that SHOULD be
```

Comment:
Why not distutils? Distlib is very different, and frankly nowhere near as significant.

Reply:
I'll switch it back.